### ISSUE TITLE
Add a placeholder to the API URL textbox

### ISSUE THREAD

**Issue by andrasfuchs** [ipfs/ipfs-webui#1807]
**Is your feature request related to a problem? Please describe.**
If the IPFS Desktop client can't connect to a previously available node, then the user needs to enter a new URL to the API. The problem is that the format of the URL is not obvious, making it difficult to guess the right text to enter.

**Describe the solution you'd like**
It would be great to have a placeholder (gray text in the empty textbox) if no value was entered as API URL.

**Describe alternatives you've considered**
The textbox could be also prefilled with the previously used URL even if that node is not reachable at the moment.

**Additional context**
Is this the right format?
![image](https://user-images.githubusercontent.com/910321/121517143-4cee8980-c9ef-11eb-87ce-5557c9cb3438.png)

If the textbox is empty, a validly formatted example URL should be displayed as the background of the textbox.
![image](https://user-images.githubusercontent.com/910321/121517228-668fd100-c9ef-11eb-86eb-43d6e7fb8ae5.png)

**Comment by lidel** [comment]
@andrasfuchs this interface is  not part of ipfs-desktop repo – i've moved this issue to ipfs-webui. 

You can find the input at https://github.com/ipfs/ipfs-webui/blob/f2bcd1c32416b56b2f4f15245cdeb5d1c4cff75a/src/components/is-not-connected/IsNotConnected.js#L82-L85   – if you open a PR with english text, I can help you with i18n later. :pray:

**Comment by andrasfuchs** [comment]
Thanks @lidel for your replies, I appreciate your help.

I started to get into this PR, but since I mainly work with Visual Studio 2019 and C# I needed to install and upgrade my Node.js / npm / Python stack. After 1.5 hours I got to the point that I still get npm errors like these:

`Error: Integrity checksum failed when using sha512: wanted sha512-Gd2UZBJDkXlY7GbJxfsE8/nvKkUEU1G38c1siN6QP6a9PT9MmHB8GnpscSmMJSoF8LOIrt8ud/wPtojys4G6+g== but got sha512-kbpaSSGJTWdAY5KPVeMOKXSrPtr8C8C7wodJbcsd51jRnmD+GZu8Y0VoU6Dm5Z4vWr0Ig/1NKuWRKf7j5aaYSg==.`

`HttpErrorGeneral: 404 Not Found - GET https://registry.npmjs.org/socket.io-client-next - Not found`

```
npm ERR! gyp ERR! find VS checking VS2019 (16.10.31321.278) found at:
npm ERR! gyp ERR! find VS "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community"
npm ERR! gyp ERR! find VS - "Visual Studio C++ core features" missing
```

The C++ feature would need 5.5 GBs of space, and I'm not sure how long the list of errors would be afterwards. 

Sorry, but I'll not spend more time on this tiny PR now, I hope you understand.
